bettermdptools/envs/binning.py

```diff
@@ -0,0 +1,105 @@
+"""
+Generated using ChatGPT.
+"""
+
+def generate_bin_edges(range_limit, n_bins, width_ratio, center=True):
+    """
+    Generates bin edges for a symmetric range with exponentially increasing/decreasing bin widths.
+
+    PARAMETERS:
+
+    range_limit {float}: 
+        The extreme value of the range [-range_limit, range_limit].
+
+    n_bins {int}:
+        Number of bins (must be odd).
+    
+    width_ratio {float}: 
+        Ratio of the outermost bin widths to the central bin width.
+
+    center {bool, default=True}: 
+        True: Outer bins are wider than the center bin.
+        False: Center bin is wider than the outer bins.
+
+    RETURNS:
+
+    bin_edges {List[float]}, shape (n_bins + 1): 
+        The edges of the bins.
+    """
```

Comment:
now that the docs PR has been merged - can you also change all of your docstrings to numpy-style?

Reply:
updated